STLYEGUIDE_SPRESS_UPDATE/gitignore

```diff
@@ -0,0 +1,8 @@
+# npm removes this file from butler on install if we name it .gitignore, but
+# it's a necessary part of the Spress template!
```

Comment:
@aCyborg does that mean you think we should rename it?

Reply:
I believe this is some comments lauren wrote. it is only marked as 'new' since it is in a new directory. this gitignore should match the git ignore in the other spress template.